nidx/nidx_text/src/lib.rs

```diff
@@ -99,7 +126,7 @@ impl TextIndexer {
     }
 
     pub fn deletions_for_resource(&self, resource: &nidx_protos::Resource) -> Vec<String> {
-        vec![resource.resource.as_ref().unwrap().uuid.clone()]
+        resource.texts_to_delete.clone()
```

Comment:
We cannot change this until after the nucliadb part is deployed that always fills in this field. Otherwise, we'll end up with duplicated fields.

So we may need to do it in two deploys.